package.json

```diff
@@ -1,6 +1,6 @@
 {
-	"name": "hm-media-weight",
-	"version": "0.1.0",
+	"name": "media-weight",
+	"private": true,
```

Comment:
What does "private" mean here? Is this just to make sure it isn't published to npm?